sdk/ai/azure-ai-inference/azure/ai/inference/prompts/__init__.py

```diff
@@ -0,0 +1,23 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+from .core import InvokerFactory
+from .core import Prompty
+
+from .renderers import MustacheRenderer
+from .parsers import PromptyChatParser
+from .utils import load
```

Comment:
We try to avoid re-exporting functions from public modules (as it becomes ambiguous as to what to `import` for developers). Making the `utils` module private would address the issue.